.gitignore

```diff
@@ -179,3 +179,5 @@ tramp
 # cask packages
 .cask/
 
+node_modules/
+TWLight/static/css/*-rtl.css
```

Comment:
We need this in version control for production deployments

Reply:
Done!